plugins/python-build/share/python-build/graalpython-20.1.0-dev

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
So that means this code isn't open source and not licensed under pyenv's normal license?

Reply:
No, it's licensed under the same license as pyenv. The phrase "All rights reserved" just clarifies from a legal perspective that all rights that are not granted under the license this file is available under are reserved by the copyright holder.

Reply:
I am not a lawyer, but as I understand it the phrase is a way to establish copyright under international law, not to restrict any licensing terms.

Reply:
If it makes it clearer, I can add the MIT license text below the copyright.

Reply:
I added the entire license text to the file, to make this clearer.

Reply:
An as an example for another project (that is not also from Oracle): https://github.com/keras-team/keras/blob/master/LICENSE

Reply:
@thedrow anything I can do to help with the issue?

Reply:
No this is fine IMO.